modules/core/src/v2/coinFactory.ts

```diff
@@ -31,6 +31,7 @@ import {
   Teos,
   Tetc,
   Teth,
+  Gteth,
```

Comment:
Have you double checked that all the `isFamily()` functions/checks don't look for the 't' prefix? Know i've seen something somewhere that evaluated`isFamilyEth()` by looking at whether the coin was `eth` or 't' + `eth`

Reply:
I've looked a bit, didn't find any. I've tested integration w/ platform and it's working well there - we have been trying to remove all such checks over time, as it's not very robust